### ISSUE TITLE
Wiki: Firefox's spell-check doesn't work

### ISSUE THREAD

**Issue by jeroenpraat** [redmatrix/hubzilla#819]
When editing a wiki-document Firefox's spell-check is disabled. When I enable this manually (using the context menu) it only looks enabled (in the context menu) for a very short period. But it still doesn't work at all.

**Comment by zotlabs** [comment]
A quick search of the web provides this helpful hint:

````
This works on Firefox:
If you add these attributes to editor's div element, Firefox will check for spelling:

contenteditable="true" spellcheck="true"
````

**Comment by git-marijus** [comment]
Does this affect markdown mimetype or bbcode mimetype or both?

**Comment by zotlabs** [comment]
I've looked - it affects markdown only when using the ace editor. I've tried the attributes I found on the web (two comments up) and these do not enable FF spellcheck in this case. There are ace spellcheck plugins, but seem to be a restricted language set and not integrated in the browser.

**Comment by jeroenpraat** [comment]
Sorry to not responding earlier. Yes, it only affects the markdown editor.

**Comment by git-marijus** [comment]
Yeah, it seems browser native spellchecking is not supported in ace editor.

**Comment by jeroenpraat** [comment]
So the use of the Ace editor causes a regression. Is it not possible to use Markdown without the Ace editor? Just like when you use bbcode for a wiki?

**Comment by git-marijus** [comment]
I guess it's a tradeoff. You would loose the markdown syntax highlighting etc. instead. There are possibilities to implement external spellcheck libs in ace. Not sure if it's worth the hassle though...
What do you think @anaqreon ?

**Comment by jeroenpraat** [comment]
We don't have syntax highlighting with BB-code either, so me personally wouldn't care (you can always switch to the preview tab (like here on Github). So to have Ace/non-Ace configurable would be a good idea I think.

**Comment by anaqreon** [comment]
It's too bad the spell checker doesn't work, but given the fact that implementing support for spell checking is not a trivial task based on the comments so far, I do not think doing so is worth the effort, at least not for me personally. Unless someone else wants to tackle this, perhaps you could find a workaround, such as copying/pasting your content temporarily into another text area that does support spell checking.

**Comment by jeroenpraat** [comment]
@anaqreon I don't ask now (based on the comments) to add spellcheck support to Ace, but to be able to *disable* Ace for markdown (like we do for bbcode).
